addons/l10n_de/i18n/de.po

```diff
@@ -4,77 +4,28 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 14.0\n"
```

Comment:
shouldn't appear. Did you rebase?

Reply:
I did rebase, but I generated new `.po` and `.pot` files in `l10n_de` since the content of the module changed quite a bit(with everything that was moved to `l10n_din5008`).
I also removed the `de.po` files in the other modules : `purchase`, `repair`, sale` and `stock`, since these modules are no longer aimed at Germany alone anymore.
That probably explains the large diff :wink: 
Do you think I should have done differently ?